scripts/gitlab/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+##############################################################################
+# Copyright (c) 2016-19, Lawrence Livermore National Security, LLC and Umpire
+# project contributors. See the COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (MIT)
+##############################################################################
+
+BUILD_DIRECTORY="build_${SYS_TYPE}_${COMPILER}"
+CCONF="host-configs/${SYS_TYPE}/${COMPILER}.cmake" 
+
+rm -rf ${BUILD_DIRECTORY} 2>/dev/null
+mkdir ${BUILD_DIRECTORY} && cd ${BUILD_DIRECTORY}
+
+cmake \
+  -C ../.gitlab/conf/${CCONF} \
+  -C ../${CCONF} \
```

Comment:
Where should build-specific feature variations go?  (-DENABLE_TOOLS, -DENABLE_FORTAN, etc?)

Reply:
- All project specific information should go into the projects host-configs directory. In other word, everything that couldn’t be shared. This includes build system variables that may not exist in other projects.
 - Tool chain configuration (typically compilers paths) can be shared, hence the creation .gitlab/conf/{conf}.

Reminder : .gitlab it to be shared among projects, but no mechanism allows it at the moment and it is easier to maintain this way, since it is still in conception phase.